packages/navigator/src/browser/navigator-tree.ts

```diff
@@ -59,18 +59,28 @@ export class FileNavigatorTree extends FileTree {
     }
 }
 
+/**
+ * File tree root node for multi-root workspaces.
+ */
 export interface WorkspaceNode extends CompositeTreeNode, SelectableTreeNode {
     children: WorkspaceRootNode[];
 }
 export namespace WorkspaceNode {
+
     export const id = 'WorkspaceNodeId';
     export const name = 'WorkspaceNode';
 
     export function is(node: TreeNode | undefined): node is WorkspaceNode {
         return CompositeTreeNode.is(node) && node.id === WorkspaceNode.id;
     }
 
+    /**
+     * @deprecated since 1.6.0 use `WorkspaceNode.create` instead.
```

Comment:
Is the name change here to clarify that a workspace node isn't necessarily the root of its true, even if it corresponds to a workspace root?

Reply:
The name is confusing: `WorkspaceNode` is the root of the file tree when in a "multi-root" workspace (potentially multiple folders) and `WorkspaceRootNode` is the node representing one of those folders part of the workspace.

With that in mind, `WorkspaceNode.createRoot` was confusing to me, instead I think it makes more sense to just `create` a `WorkspaceNode`. It being used as a root is implied by design. I added documentation on the different node types to help clear confusion because this is what led me to write wrong logic.